__tests__/integration.test.js

```diff
@@ -32,3 +32,40 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET:200 responds with a single article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
+        expect(body.article.author).toBe("butter_bridge");
+        expect(body.article.title).toBe("Living in the shadow of a great man");
+        expect(body.article.article_id).toBe(1);
+        expect(body.article.body).toBe("I find this existence challenging");
+        expect(body.article.topic).toBe("mitch");
+        expect(body.article.created_at).toBe("2020-07-09T20:11:00.000Z");
+        expect(body.article.votes).toBe(100);
+        expect(body.article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
```

Comment:
The test for this endpoint is great, you could use a flexible jest matcher here. Here's the link for reference: https://jestjs.io/docs/expect#tomatchobjectobject